package.json

```diff
@@ -8,7 +8,6 @@
     "watch": "ng build --watch --configuration development",
     "test": "ng test"
   },
-  "private": true,
```

Comment:
does this cause any warnings about required fields to print when you run npm install?
if it's a package you'll publish, the CLI is stricter about what fields this file should have.

Reply:
I don't think there were any about required fields, just a bunch about deprecated packages. I previously used a npm publish task for Azure Artifacts and it would not run unless this line was taken out. Since I'm not using that task anymore I put it back the way it was.